frontend/src/app/models/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface User{
+    firstName: string,
+    lastName: string,
+    email: string,
+    picture: string
+}
```

Comment:
Add empty line to the bottom of this file

Reply:
Done